docs/hcc.md

```diff
@@ -1,42 +1,105 @@
-# git-source-control for Health Connect Cloud Overview
-
-## Development Workflow
-
-git-source-control has two main use modes: Basic and Expert. For Health Connect Cloud, we recommend beginning with Basic Mode, since it includes all of the necessary functionality for proper source control development while being easy to use for those with less Git experience. The recommended workflow for Health Connect Cloud is to create interface branches, create merge requests into development, and then pull the changes into the main dev development namespace.
-
-A Health Connect Cloud Dev deployment will have a main namespace (which is also present in Test and Prod) and should also have a dedicated namespace for each developer contributing. These namespaces can be configured via the Cloud Portal. It is important for changes to originate in developer-specific namespaces; the main namespace in the dev deployment should always be kept up to date with the development branch in GitLab, either through a CI/CD pipeline or by use of the Git Pull page from git-source-control.
+# Change Control with Embedded Git for Health Connect Cloud
+
+## General 
+The recommended workflow for Health Connect Cloud is to: create an interface branch, make changes, sync them to GitLab, create a merge request, review and approve the merge request, and then pull the changes into a protected namespace.
+
+Step 1: Create an interface branch based on live
+![Step 1: Create a feature branch based on live](images/hcc/hcc-step-1.png)
+
+Step 2: Make changes in the User Namespace
+
+Step 3: Sync to get latest from live and push to GitLab
+![Step 3: Sync to get latest from live and push to GitLab](images/hcc/hcc-step-3.png)
+
+Step 4: Merge request and repeat 2-3
+![Step 4: Merge request and repeat 2-3](images/hcc/hcc-step-4.png)
+
+A Health Connect Cloud Dev deployment will have at least one Protected Namespace and should also have a dedicated User Namespace for each user. These namespaces can be configured via the Cloud Portal for the selected Health Connect Cloud deployment. It is important for changes to originate in User Namespaces. Protected Namespaces in the Dev deployment are automatically kept up to date with the development branch of the associated GitLab repo through a CI/CD pipeline.
+
+# Prerequisites: InterSystems Responsibilities
+ - Create the GitLab repository for the organization
+ - Grant appropriate access and privileges to the GitLab repository
+ - Onboard customers on source control tools for Health Connect Cloud
+ - Maintain change control documentation and videos
+   - [Official Change Control Documentation](https://docs.intersystems.com)
```

Comment:
Can we somehow comment this out for now? (Until we have more official documentation to point to more specifically?)